Dockerfile

```diff
@@ -26,10 +26,12 @@ COPY packages/transition-backend packages/transition-backend
 COPY packages/transition-frontend packages/transition-frontend
 RUN yarn install
 
-
 # Copy the rest. (node_modules are excluded in .dockerignore)
 COPY . /app
 
+# Setup the example as a default configuration for the image
+COPY .env.docker /app/.env
```

Comment:
Why not use the .env.example file and just sed the lines that needs to be different for docker?
Or
we could use ENV entries in the Dockerfile to override those entries with other values.